apim-lab/6-analytics-monitoring/analytics-monitoring-6-2-application-insights.md

```diff
@@ -12,7 +12,7 @@ Azure API Management allows for easy integration with Azure Application Insights
 
 ### Create an Azure Application Insights instance
 
-Before you can use Azure Application Insights, you first need to create an instance of the service.
+Before you can use Azure Application Insights, you first need to create an instance of the service. The naming convention for Application Insights is: `appi-<environment>-<region>-<application-name>-<owner>-<instance>`

```

Comment:
Thank you for pointing out naming convention repeatedly. This is great. You could also always link `naming convention` to the common abbreviations: https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

Reply:
The link is already in the naming convention section.